apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/page/PageStateChanger.java

```diff
@@ -49,6 +72,49 @@ public PageStateResponse savePageAsDraft(Long id) {
         }
         return response;
     }
+    
+	@Transactional
+	public PageStateResponse deletePageDraft(Long id) {
+		PageStateResponse response = new PageStateResponse();
+		try {
+			Optional<WaTemplate> result = templateRepository.findById(id);
+			if (result.isPresent()) {
+				WaTemplate page = result.get();
+				if (!page.getTemplateType().equals(PageConstants.PUBLISHED_WITH_DRAFT)) {
+					throw new PageUpdateException(PageConstants.DRAFT_NOT_FOUND);
+				}
```

Comment:
should this be:
```java
if (!page.getTemplateType().equals(PageConstants.PUBLISHED_WITH_DRAFT) || !page.getTemplateType().equals(PageConstants.DRAFT)) {
...
}
```

to allow deleting a Draft that has not yet been published?

Reply:
Hi @mpeels  take a look. Thanks.